src/containers/execution-environment-list/execution_environment_list.tsx

```
@@ -476,15 +473,8 @@ class ExecutionEnvironmentList extends React.Component<
                 () => this.queryEnvironments(),
               );
             })
-            .catch((err) => {
-              this.setState({
-                formError: err.response.data.errors.map((error) => {
-                  return {
-                    title: error.title,
-                    detail: error.source.parameter + ': ' + error.detail,
-                  };
-                }),
-              });
+            .catch((e) => {
+              console.log(e);
             });
```

Empty catch with forgotten console.log?

```suggestion
```

---

All errors are handled in previous promises, this would fail on unexpected error (in setState), so I will show something like Unexpected error alert with error message.

---

So, this is problem. This catch pops up whenever form is invalid, so it does really nothing, because those errors are displayed in the RepositoryForm component. With alert message, the alert message is displayed whenever forms throw an error (aditionaly to its own error handling) and shows alert behind the form modal, which is bad. I will delete the catch for now and we can discuss it later.